anti.md

```diff
@@ -58,12 +93,24 @@ making language design decisions), as opposed to "systems" aspects of the
 project (e.g., implementing a complicated semantics that doesn't require a lot
 of language design)?_
 
+Very little of their time would be spent on language design because writing in
+the language is the smallest part of the user experience. A ton of time would
+be necessary to parse websites and figure out how to get all of their different
+types of data and updates together.
 
 ### Scope
 _How big an idea is this? How ambitious is this project?_
 
+This is a scalable project. Since the language design aspect is so easy and
+almost all of the time would be devoted to adding website support, the project
+could be toned down or up by just supporting more or fewer websites.
```

Comment:
Depending on how often sites change their formatting, even just maintenance could be a big problem, so there's that.  Except maybe for those sites which provide APIs.